### PR TITLE
Adding a ROADMAP.md

### PR DESCRIPTION
**Pull request by vishnuchalla**
## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] Optimization
- [x] Documentation Update

## Description
Adding a ROADMAP.md

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

**Review by rsevilla87 — APPROVED** [review]
lgtm

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Kube-Burner Roadmap
+
+This document is meant to define high-level plans for the kube-burner project and is neither complete nor prescriptive. Following are a list of enhancements that we are planning to work on adding support to our application. Each and every action item is tracked using github [issues](https://github.com/cloud-bulldozer/kube-burner/issues).
+
+
+* [[RFE Enhancement]](https://github.com/cloud-bulldozer/kube-burner/issues/409) Funtionality to compare kube-burner collected metrics to detect regressions automatically.
```
[the remark this answers is not in the document]
Comment (vishnuchalla) [reply ROADMAP.md]: Yes, Good idea. Changed as suggested. Thank you!

**Review by jtaleric — APPROVED** [review]
lgtm!

**Review by smalleni — APPROVED** [review]
LGTM

**Pull request merged**